### DISCUSSION TITLE
Support for PMTile?

### DISCUSSION THREAD

**Discussion by sid-kap** [Toblerity/Fiona#1336] [Q&A]
Are there any plans to support reading and writing the [PMTile format](https://docs.protomaps.com/pmtiles/), now that [GDAL can read and write PMTile](https://gdal.org/drivers/vector/pmtiles.html)?

**Comment by sgillies** [comment]
Yes, I expect we'll have basic support for Protomaps tiles in 1.10.0. Full parity with [its native tools](https://github.com/protomaps/PMTiles?tab=readme-ov-file#creating-pmtiles) may be beyond this project.
